File: android/app/src/main/java/io/ionic/starter/data/models/Weather.kt
package io.ionic.starter.data.models

import java.io.Serializable

data class Weather(
    var currentTemperature: String? = null,
    var currentFeelLikeTemp: String? = null,
    var windSpeed: String? = null,
    var _24hHighTemp: String? = null,
    var _24LowTemp: String? = null,
    var _24WindSpeed: String? = null,
    var _24WindDirection: String? = null,
    var _24Precip: String? = null,
    var _24PrecipProb: String? = null,
    var weatherSymbol : Int = 0,
    var _6hoursForeCastTemp : List<DateValue>? = emptyList(),
    var _6hoursForeCastIcons : List<DateValue>? = emptyList(),

    var _5DaysForeCastIcons : List<DateValue>? = emptyList(),
    var _5DaysForeCastHighTemp : List<DateValue>? = emptyList(),
    var _5DaysForeCastLowTemp : List<DateValue>? = emptyList(),

    ):Serializable

File: ios/App/App/data/models/Weather.swift
//
//  Weather.swift
//  App
//
//  Created by Shahzaib Ali on 01/08/2024.
//

import Foundation
struct Weather {
    var currentTemperature: String? = nil
    var currentFeelLikeTemp: String? = nil
    var windSpeed: String? = nil
    var cityName: String = ""
    var _24hHighTemp: String? = nil
    var _24LowTemp: String? = nil
    var _24WindSpeed: String? = nil
    var _24WindDirection: String? = nil
    var _24Precip: String? = nil
    var _24PrecipProb: String? = nil
    var weatherSymbol: Int? = 0
    var _6hoursForeCastTemp: [WeatherDate]? = nil
    var _6hoursForeCastIcons: [WeatherDate]? = nil
    var _5DaysForeCastIcons: [WeatherDate]? = nil
    var _5DaysForeCastHighTemp: [WeatherDate]? = nil
    var _5DaysForeCastLowTemp: [WeatherDate]? = nil
    
}
